src/installation/guides/fde.md

```diff
@@ -1,132 +1,131 @@
 # Full Disk Encryption
 
-**Warning**: Your drive's block device and other information may be different,
-so make sure it is correct.
+**Warning**: This guide uses `/dev/vda` as the example device name, Your drive's block device and other information may be different, so make sure it is correct.
```

Comment:
I changed sda to vda in a effort to make people pay closer attention to their drive's block device information, 
But i now think the better thing to do would be to change it to /dev/xxx1 or /dev/xxx2 instead of just droping it altogether.

Reply:
We aren't here to save people from themselves when following a destructive guide that describes optional and advanced configuration. The example commands should be generally meaningful , not have bogus placeholders that require replacement.